### DISCUSSION TITLE
Help For new function on v 3.11.0

### DISCUSSION THREAD

**Discussion by juankstylez** [nutgram/nutgram#293] [General]
Intentando registrar un nuevo comando ! no logro conseguir mi error y en la documentación no hay nada de informacion ! me guie a través de un comentario respondido a mi ! :( 

1 -> https://prnt.sc/fuAq9bmJ7xRU  My Routes

2 -> https://prnt.sc/-46c_P6Hs_Rn My controller

3-> https://prnt.sc/wHgRezC4eV4W My error ! :( 

Helpme please !

**Comment by Lukasss93** [comment]
Hello!
The documentation will be updated soon, you can see the PR here: https://github.com/nutgram/docs/pull/14
TLDR: you need to use: `registerCommand` method

PS: next time speak english 😉

**Reply by juankstylez** [reply]
Thanks, this worked for me! <3

**Comment by Lukasss93** [comment]
Updated! https://nutgram.dev/docs/usage/keyboards
